int64 direction
---
std_msgs/Float32MultiArray stateVec
bool done
int64 steps